### ISSUE TITLE
Index Error in loss_fn function

### ISSUE THREAD

**Issue by thomasarmstrong98** [ishaqadenali/SVRG-Pytorch#1]
I have tried to run this code as you created it, where the data is loaded in rather than from pytorch's dataloader. 

When running it however, I am getting the following error:

"Dimension out of range (expected to be in range of [-1, 0], but got 1)"

This is referring to line 70 of the MNIST example script which is the following 

"    loss = loss_fn(y_pred, torch.max(yt, 1)[1]) "

Any idea on why I am getting, I have tried debugging but with no luck! Thanks

**Comment by thomasarmstrong98** [comment]
I believe it originates from the yt variable, as torch.max(yt, 1) is giving the error. For my case, this yt is a tensor, where has it gone wrong?
